iceoryx_hoofs/include/iceoryx_hoofs/cxx/variant.hpp

```diff
@@ -277,14 +275,15 @@ class variant
   private:
     template <typename T>
     bool has_bad_variant_element_access() const noexcept;
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1 : internal convenience function to easily use IOX_LOG
     static void error_message(const char* source, const char* msg) noexcept;
```

Comment:
I think this method could also be removed and refactored to use `IOX_LOG` directly?